src/odemis/acq/stream/_helper.py

```diff
@@ -669,6 +669,41 @@ def __init__(self, name, detector, dataflow, emitter, analyzer=None, **kwargs):
 
     # onActive & projection: same as the standard LiveStream
 
+    # overwrite fct _is_active_setter in _base.py
```

Comment:
Technically, it's called "overriding". I'd rephrase the comment to something like:
`Override Stream._is_active_setter() in _base.py`